lib/ui/widget/dimmed_image_background.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+
+import 'blurred_background.dart';
+import 'dimmed_background.dart';
+
+class DimmedImageBackground extends StatefulWidget {
+  final ImageProvider image;
+  final bool shouldBlur;
+  final bool shouldAnimate;
+  final Duration animationDuration;
+
+  const DimmedImageBackground({
+    super.key,
+    required this.image,
+    this.shouldBlur = false,
+    this.shouldAnimate = false,
+    this.animationDuration = const Duration(milliseconds: 0),
+  });
+
+  @override
+  DimmedImageBackgroundState createState() => DimmedImageBackgroundState();
+}
+
+class DimmedImageBackgroundState extends State<DimmedImageBackground>
+    with TickerProviderStateMixin {
+  late final AnimationController _backgroundBlurAnimationController =
+      AnimationController(
+    duration: widget.animationDuration,
+    vsync: this,
+  );
+  late final Animation<double> _backgroundBlurAnimation = CurvedAnimation(
+    parent: _backgroundBlurAnimationController,
+    curve: Curves.easeIn,
+  );
+
+  @override
+  Widget build(BuildContext context) {
+    if (widget.shouldAnimate) _backgroundBlurAnimationController.forward();
```

Comment:
Does it show the background & blur if we pass `shouldAnimate` with `false`? 🤔

Reply:
@markgravity here's the output with `false`.

https://github.com/nimblehq/ic-flutter-avishek/assets/8093908/26c44cc3-7c04-4082-ac24-f5b89a2e8202

Reply:
Can we make it show the background & blur if the `shouldAnimate` is `false`?
Then we can re-use it on another screen like the Reset Password screen if we plan to implement it 🙏 

![CleanShot 2023-07-11 at 13 42 11](https://github.com/nimblehq/ic-flutter-avishek/assets/17875522/9c8a6714-a982-4bee-8507-9f6c1032fbab)

Reply:
@markgravity we can just pass `const Duration(milliseconds: 0)` for the animation duration. It gives the same result. 🤔